src/tests/multihost/alltests/test_kcm.py

```diff
@@ -52,3 +52,36 @@ def test_client_timeout(self, multihost, backupsssdconf):
                                                    " /var/log/sssd/"
                                                    "sssd_kcm.log")
         assert 'Terminated client' in grep_cmd.stdout_text
+
+    def test_kcm_check_socket_path(self, multihost, enable_kcm):
+        """
+        @Title: kcm: Test socket path when sssd-kcm is activated by systemd
+        #https://github.com/SSSD/sssd/issues/5406
+        """
+        # Start from a known-good state after removing log file and adding a
+        # new socket path
+        multihost.master[0].service_sssd('stop')
+        self._stop_kcm(multihost)
+        self._remove_kcm_log_file(multihost)
```

Comment:
You can use remove_sssd_cache function
https://github.com/SSSD/sssd/blob/master/src/tests/multihost/sssd/testlib/common/utils.py#L380

Reply:
As I'm deleting the kcm log file shouldn't I use delete_sssd_domain_log() instead?

Reply:
delete_sssd_domain_log()  will remove sssd_<domainname>.log file as it only take domainname as argument. To remove other files we can specify path in remove_sssd_cache()

Reply:
That makes sense